test/om-linker/Inputs/a.mlir

```diff
@@ -1,5 +1,15 @@
 module {
+  hw.hierpath private @xmr [@M1::@s1, @M2]
   om.class @A(%arg: i1) {
   }
   om.class @Conflict(){}
+  hw.module private @M2() {}
+  hw.module @M1(in %a: i1) {
+
+  sv.always posedge %a {
+    sv.if %a {
+      hw.instance "" sym @s1 @M2() -> ()
```

Comment:
nit: I'm surprised that this is a valid IR. TBH we shouldn't allow instances in procedural regions so can we change the test something different (e.g. sv.ifdef instead?)